lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,65 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+      )
+
+      let :issue, object(
+        snippet: string?,
+      )
+    end
+
+    register_config_schema SCHEMA.config
+
+    DEFAULT_TARGET = "".freeze
+
+    def self.config_example
+      <<~'YAML'
+        root_dir: project/
+        target:
+          - myworkflow.yml
+          - workflows/anotherworflow.yml
```

Comment:
[suggestion] It may be less confusing to clariy `.github/workflows/`:

```suggestion
          - .github/workflows/one_worflow.yml
          - .github/workflows/another_worflow.yml
```

Reply:
I wanted to express that any files can be selected, not just in the `.github` directory, but I guess it makes sense to put it into the context of the `.github/workflows/` standard location.

Reply:
I see. It seems good to add an example of `.github/workflows/` to the existing examples like `myworkflow.yml`. 👍🏼

Reply:
I changed the example to your original suggestion. Looks like actionlint supports neither globs, nor directories as the target file parameters. Only list of files, or omit the files argument completely